Formula/govendor.rb

```diff
@@ -0,0 +1,36 @@
+class Govendor < Formula
+  desc "Go vendor tool that works with the standard vendor file."
+  homepage "https://github.com/kardianos/govendor"
+  url "https://github.com/kardianos/govendor/archive/v1.0.3.tar.gz"
+  sha256 "7d7a032355ee4117822150db72a9db745d9a147a2e32972e58a65fe7545444f2"
+  head "https://github.com/kardianos/govendor.git"
+
+  depends_on "go"
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GOOS"] = "darwin"
+    ENV["GOARCH"] = MacOS.prefer_64_bit? ? "amd64" : "386"
+
+    (buildpath/"src/github.com/kardianos/").mkpath
+    ln_sf buildpath, buildpath/"src/github.com/kardianos/govendor"
+    system "go", "build", "-o", bin/"govendor"
+  end
+
+  test do
+    # Default HOMEBREW_TEMP is /tmp, which is actually a symlink to /private/tmp.
+    # `govendor` bails without `.realpath` as it expects $GOPATH to be "real" path.
+    ENV["GOPATH"] = testpath.realpath
+    (testpath/"src/github.com/project/testing").mkpath
```

Comment:
I personally find our `ln_sf` style as clear as mud in terms of what is getting executed where & when, at a glance.